pkg/instancegroups/instancegroups.go

```diff
@@ -227,11 +201,47 @@ func (r *RollingUpdateInstanceGroup) RollingUpdate(rollingUpdateData *RollingUpd
 				rollingUpdateData.Interactive = false
 			}
 		}
+
+	sweep:
+		for concurrency > 0 {
```

Comment:
I was pondering whether we needed this "sweep" block.  I think in most of these loops we wouldn't, except that we avoid some validate calls by having it here.  Is that right?  If so ... could we have a comment to save our future selves? :-)

Reply:
This allows us to keep the concurrency up. For now, validation will likely start returning failures once one node starts draining until its replacement is fully ready, so if we don't sweep we won't be able to sustain more than one or two concurrent drains.

Reply:
Added a comment